src/Http/Response.php

```diff
@@ -46,6 +46,10 @@ class Response implements ResponseInterface
 {
     use MessageTrait;
 
+    const MIN_STATUS_CODE_VALUE = 100;
+
+    const MAX_STATUS_CODE_VALUE = 599;
```

Comment:
Usually you would group constants:

- STATUS_CODE_MAX
- STATUS_CODE_MIN

Reply:
Agreed. I can fix it up.

Reply:
You would also have to do that in master now.